modules/farm_fd2_school/src/entrypoints/HTML_CSS/App.vue

```diff
@@ -3,7 +3,86 @@
     id="HTML-CSS"
     data-cy="HTML-CSS"
   >
-    <p>Add HTML for Harvest form here.</p>
+    <p>Harvest</p>
+    <label for="start">Date: </label>
+    <input
+      type="date"
+      id="start"
+      name="trip-start"
+      value="2019-06-15"
+      min="2019-01-01"
+      max="2019-12-31"
+    />
+    <br />
+
+    <label for="crop">Crop: </label>
+    <select>
+      <option>RADISH</option>
```

Comment:
It would be better to use the `selected` attribute here rather than reordering the options in the list.